Classes/Controller/ModFuncController.php

```diff
@@ -67,7 +67,8 @@ public function modMenu()
         $languages = [];
         $languageRecords = $this->getDatabaseConnection()->exec_SELECTgetRows('uid,title', 'sys_language', 'hidden=0', '', 'title');
         if (is_array($languageRecords) && !empty($languageRecords)) {
-            $languages[] = 'Default';
+            $defaultLanguageLabel = BackendUtility::getModTSconfig($this->pObj->id, 'mod.SHARED.defaultLanguageLabel');
+            $languages[] = $defaultLanguageLabel['value'] ? $defaultLanguageLabel['value'] : 'Default';
```

Comment:
Instead of `$defaultLanguageLabel['value'] ? $defaultLanguageLabel['value'] : 'Default';` you also could write

```
$languages[] = $defaultLanguageLabel['value'] ?:  'Default';
```